components/radio/stories/radio.stories.js

```diff
@@ -130,11 +130,15 @@ Disabled.parameters = {
 };
 
 /**
- * A radio group has a read-only option for when it's in the disabled state but still needs to be shown.
- * This allows for content to be copied, but not interacted with or changed.
+ * A radio group has a read-only option for when it's functionally disabled but still needs to be shown.
+ * This allows for label content to be copied, but prevents the input from being interacted with or changed.
  *
- * - Read-only radio buttons are disabled, but not all disabled radio buttons are read-only.
- * - Read-only radio buttons do not have a focus ring, but the button should be focusable.
+ * Read-only radio buttons:
+ * - prevent interaction like disabled, but not all disabled radio buttons are read-only
+ * - are immutable, i.e. their checked state cannot be changed
+ * - are keyboard focusable and communicate state to assistive technology
+ * - use `aria-disabled` since the `readonly` attribute [is not valid](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/readonly#overview)
+ * - use `preventDefault()` on click to prevent changing the selection via pointer or keyboard interaction
```

Comment:
```suggestion
 * Read-only radio buttons:
 * - prevent interaction like disabled, but not all disabled radio buttons are read-only
 * - are immutable, i.e. their checked state cannot be changed
 * - are keyboard focusable and communicate state to assistive technology
 * - use `aria-disabled` since the `readonly` attribute [is not valid](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/readonly#overview) and `aria-readonly` is not currently announced by the majority of screen readers.
```
What do you think about leaving out the last sentence since we don't necessarily need to prescribe the how, and that is covered under the "immutable" bullet point?

I could also see dropping the first bullet point that originates from the original docs, as it feels a little confusing and not totally necessary? I'm on the fence about that one and don't have a problem with keeping it.

Reply:
I guess I was trying to emphasize that it required scripting to accomplish and wasn't an inherent feature, which is different behavior than applying `readonly` to other input types, but if WC pulls in the change we can confirm.